app/components/Activity/Activity.js

```diff
@@ -10,15 +12,31 @@ import X from 'components/Icon/X'
 import messages from './messages'
 import ActivityListItem from './ActivityListItem'
 
+const StyledList = styled(List)`
+  outline: none;
+  padding-right: 45px;
```

Comment:
Can we do this using our theme spacing units rather than custom values? You could add the styled-system spacing props onto this element like so:

```js
import styled from 'styled-components'
import { space } from 'styled-system'

const StyledList = styled(List)(space)
```

Which will allow you to use all of our themes predefined spacing properties rather than custom units. (`<StyledList  pr={5}/>` would be equivalent to `padding-right: 45px;` for example), and would allow those props to be applied by the caller rather than being preset on the element definition.

See https://github.com/jxnblk/styled-system